guidelines.md

```diff
@@ -82,9 +82,10 @@ Some errors may also contain additional information relevant to the error on top
 ## Authentication
 
 Each Mews environment (e.g. demo, production) requires a different set of tokens. Please use the tokens below to connect to the Mews Demo environments. To sign into the system, use the following credentials:
-* **PlatformAddress** - https://demo.mews.li
+* **PlatformAddress** - `https://api.mews-demo.com`
+* **WebSocketPlatformAddress** - `https://ws.mews-demo.com`
```

Comment:
```suggestion
* **WebSocketPlatformAddress** - `wss://ws.mews-demo.com`
```

Reply:
No, this is the platform address.  no websocket URL. Depends on how we will define it here.